datafusion/src/physical_plan/windows.rs

```diff
@@ -61,24 +63,27 @@ pub struct WindowAggExec {
 /// Create a physical expression for window function
 pub fn create_window_expr(
     fun: &WindowFunction,
+    name: String,
     args: &[Arc<dyn PhysicalExpr>],
+    _partition_by: &[Arc<dyn PhysicalExpr>],
+    _order_by: &[PhysicalSortExpr],
+    _window_frame: WindowFrame,
```

Comment:
nit: once this is merged, it would be cool to add a link to this code in the respective issues that are tracking these missing features to make it easier for new contributors to start contributing.

Reply:
I just realized that this PR changes the behavior when there is a non-empty partition_by/order_by/window_frame: before (at least in ballista) it would error, whereas now it is silently ignored. Maybe it is worth erroring is they aren't empty to make it explicit that there is no support?

Reply:
it'll be further developed in https://github.com/apache/arrow-datafusion/pull/520

Reply:
but that's a good point let me add guard here.

Reply:
this is fixed now